Automattic/jetpack
# CLI: Refactor jetpack changelog command to pass directly to changelogger

## Proposed changes:

This refactor lets us massage the changelog entries for the monorepo via the CLI, instead of just calling the changelogger in interactive mode. This will let us add any custom questions or data we want without interfering with the public package. 

This doesn't add the new prompt for if it's a user facing change, as discussed in https://github.com/Automattic/jetpack/pull/34058

It just lays the groundwork to do that.
### Other information:


## Jetpack product discussion


Internal ref: p1699566741898429-slack-C05Q5HSS013

## Does this pull request change what data or activity we track or use?


No
## Testing instructions:






Make sure the `jetpack changelog add` command still works as expected:

- `jetpack changelog add <type>/<project> should add a changelog file to the specified project
- If you make a change to a project, just running `jetpack changelog add` should find the project and let you add a changelog file to it.
- If you make changes to multiple projects, you should have the option to add the same changelog file to all of them
- It should give the appropriate changelog types for projects with specified changelog types, like jetpack.
